### DISCUSSION TITLE
Mod库：DokiChineseMods

### DISCUSSION THREAD

**Discussion by mtr-static-official** [imgradeone/revolution#15] [DDLC 相关网站投稿 / DDLC Related Website]
这次还是我搭建的网站，属于Monika Forum的姊妹站。
https://mods.ddlc.love/
和Monika Forum共享用户数据库_（在2022.2.21 15点前注册的用户可以直接登陆）_与Mod库，并支持DCC Auth验证服务登录
相较于Monika Forum，DokiChineseMods：

- Mod分类更加详尽
- 更便利的查找Mods
- 纯净的访问体验
@imgradeone

**Comment by imgradeone** [comment]
您投稿的是链接，我将会把它移到 [DDLC 相关网站投稿 / DDLC Related Website](https://github.com/imgradeone/revolution/discussions/categories/ddlc-%E7%9B%B8%E5%85%B3%E7%BD%91%E7%AB%99%E6%8A%95%E7%A8%BF-ddlc-related-website) 分类
